lua/examples/policy.lua

```diff
@@ -206,6 +228,13 @@ local function lookup_lpm_policy(pkt_info)
 	if pkt_info.inner_ip_ver == policylib.c.IPV6 then
 		local ipv6_hdr = ffi.cast("struct rte_ipv6_hdr *",
 			pkt_info.inner_l3_hdr)
+
+		local dst_addr = ffi.cast("struct in6_addr &",
+			ipv6_hdr.dst_addr)
+		if ipv6_addrs_equal(dst_addr, address_ipv6_ex) then
```

Comment:
This would be true if `ipv6_hdr.dst_addr` were a `struct in6_addr`, but it's actually a `uint8[16]`, so we have to cast it. The fact that it's defined as a byte array is a reflection of how `struct rte_ipv6_hdr` is defined in DPDK.

Reply:
Thank you for the feedback.